packages/client/src/base-client.ts

```diff
@@ -67,6 +69,18 @@ export class BaseClient {
     return await this.connection.withDeadline(deadline, fn);
   }
 
+  /**
+   * Set an {@link https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal | `AbortSignal`} that, when aborted,
+   * cancels any ongoing requests executed in `fn`'s scope.
+   *
+   * @returns value returned from `fn`
+   *
+   * @see {@link Connection.withAbortSignal}
+   */
+  async withAbortSignal<ReturnType>(abortSignal: AbortSignal, fn: () => Promise<ReturnType>): Promise<ReturnType> {
```

Comment:
Are you thinking that there's an argument for entire words over single letter type parameters in places where the type will be displayed to users?